validator-rust/src/components/exponential_mechanism.rs

```diff
@@ -151,6 +135,26 @@ impl Expandable for proto::ExponentialMechanism {
             // this case should never happen
         } else { return Err(Error::from("Variant must be defined")) }
 
+        if privacy_definition.protect_sensitivity || !properties.contains_key(&IndexKey::from("sensitivity")) {
```

Comment:
Is it acceptable for this condition to fail if there is a value set for sensitivity, but protect_sensitivity is enabled?

Reply:
You're right, if there is a user-defined sensitivity, it would be better to error if `protect_sensitivity` is enabled. Will update.